dataplane-webserver/tests/integration_test.rs

```diff
@@ -1,155 +1,69 @@
-
-        let app = test::init_service(
-            App::new()
-                .app_data(web::Data::new(cfg.clone()))
-                .app_data(web::Data::new(http_client.clone()))
-                .service(web::scope("/{namespace}/metrics").service(metrics::query_range)),
-        )
-        .await;
-
-        let url = "/org-coredb-inst-control-plane-dev/metrics/query_range";
-        let query = "(sum by (namespace) (max_over_time(pg_stat_activity_count{namespace=\"org-coredb-inst-control-plane-dev\"}[1h])))";
-        // 24 hours ago, unix time
-        let start = SystemTime::now()
-            .duration_since(SystemTime::UNIX_EPOCH)
-            .expect("Failed to get UNIX time")
-            .as_secs()
-            - 24 * 60 * 60;
-        let query_url = format_prometheus_query(url, query, start);
-        let req = test::TestRequest::get()
-            .uri(query_url.as_str())
-            .to_request();
-        let resp = test::call_service(&app, req).await;
-        assert!(resp.status().is_success());
+use actix_web::{test, web, App, Error};
+use dataplane_webserver::config;
+use dataplane_webserver::routes::secrets::{get_secret_v1, update_postgres_password, get_secret_names_v1};
+use serde_json::json;
+use actix_http::Request;
+
+async fn create_test_app() -> impl actix_web::dev::Service<actix_http::Request, Response = actix_web::dev::ServiceResponse, Error = actix_web::Error> {
+    let cfg = config::Config::default();
+    let http_client = reqwest::Client::new();
+
+    test::init_service(
+        App::new()
+            .app_data(web::Data::new(cfg))
+            .app_data(web::Data::new(http_client))
+            .service(
+                web::scope("/api/v1/orgs/{org_id}/instances/{instance_id}")
+                    .service(get_secret_names_v1)
+                    .service(get_secret_v1)
+                    .service(update_postgres_password)
+            )
+    ).await
+}
 
-        // 25 hours ago, unix time
-        let start = SystemTime::now()
-            .duration_since(SystemTime::UNIX_EPOCH)
-            .expect("Failed to get UNIX time")
-            .as_secs()
-            - 25 * 60 * 60;
-        let query_url = format_prometheus_query(url, query, start);
-        let req = test::TestRequest::get()
-            .uri(query_url.as_str())
-            .to_request();
-        let resp = test::call_service(&app, req).await;
-        // It should be a client error to request greater time frame than 1 day
-        assert!(resp.status().is_client_error());
+#[actix_web::test]
+async fn test_get_secret_names_v1() {
+    let app = create_test_app().await;
+
+    let req = test::TestRequest::get()
+        .uri("/api/v1/orgs/org_2T7FJA0DpaNBnELVLU1IS4XzZG0/instances/inst_1696253936968_TblNOY_6/secrets")
+        .to_request();
+    
+    let resp = test::call_service(&app, req).await;
+
+    assert!(resp.status().is_success(), "Response status is not successful: {:?}", resp.status());
```

Comment:
```suggestion
            let status_code = resp.status();
            if status_code.is_success().not() {
                let bytes = test::read_body(resp).await;
                let msg = str::from_utf8(&bytes).unwrap();

                panic!("Request failed: {} - {msg}", status_code.as_u16())
            }
```

I'd suggest something like this so we know what the actual error is